### PR TITLE
Adding Travis CI

### PR DESCRIPTION
**Pull request by darkodosenovic**
* spec_helper was missing the infer test from location making the test
    fail because rspec did know what kind of test to run.
    * Fix some failing test after changes.
    NOTE:
    * Team should start adding test for changes now on. the only test
    coverage are for the helpers. there should maybe be test for navigation,
    etc.

### CODE REVIEW

**Comment by MathieuGilbert** [comment]
:hankey:  travis fails

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: ruby
```
Comment (jbabe) [inline .travis.yml]: `sudo: false` will make it run faster

http://docs.travis-ci.com/user/workers/container-based-infrastructure/

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant: 
Although I don't see the point in adding Travis without fixing the tests...

**Comment by jbabe** [comment]
me either - I think we need to fix them

**Comment by rubene** [comment]
:wave:

**Comment by MathieuGilbert** [comment]
:hankey: :+1: 
Coverage (83.87%) is below the expected minimum coverage (100.00%)

**Comment by rubene** [comment]
@MathieuGilbert writing all the tests

**Comment by rubene** [comment]
:wave:

File: spec/ama_layout/decorators/moneris_decorator_spec.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
+describe AmaLayout::MonerisDecorator, type: :decorator do
+  let(:moneris) { AmaLayout::Moneris.new }
+  let(:moneris_presenter) { moneris.decorate }
+
+  describe "#textbox" do
+    it "return textbox style file" do
+      expect(moneris_presenter.textbox).to include "-webkit-appearance: none;"
```
Comment (MathieuGilbert) [inline spec/ama_layout/decorators/moneris_decorator_spec.rb]: :eyes:

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant: so many tests

**Comment by rubene** [comment]
:wave: gem version

**Comment by darkodosenovic** [comment]
:elephant: :elephant: :elephant:

**Pull request merged**